_layouts/category.html

```diff
@@ -13,7 +13,13 @@
 <div class="grid-x grid-padding-x pattern-category-title">
   <div class="wrapper">
     <div class="small-12 cell">
-      <a href="#"><h2>{{ page.title }}<span class="category-counter">({{ patterns.size }})</span></h2><img src="/images/down.svg" /></a>
+      <a id="content" aria-expanded="false" aria-describedby="category-menu-tip" href="#">
```

Comment:
Really pleased about this change. Should make a big difference to the ease of navigation with VO.